packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -969,7 +999,24 @@ function runTest<TChannelFactory extends IChannelFactory, TOperation extends Bas
 ): void {
 	const itFn = options.only.has(seed) ? it.only : options.skip.has(seed) ? it.skip : it;
 	itFn(`seed ${seed}`, async () => {
-		await runTestForSeed(model, options, seed, saveInfo);
+		try {
+			await runTestForSeed(model, options, seed, saveInfo);
+		} catch (error) {
+			if (!saveInfo) {
```

Comment:
It's a bit silly to do this extra work in CI. To avoid that, could you put a `process.env.TF_BUILD` check on `saveInfo` and overwrite it in CI environments? That way we don't write the original file either :)

example from a recent PR here: packages/test/snapshots/src/testContent.ts

Reply:
lol, didn't notice github didn't copy the whole PR and just the filename: https://github.com/microsoft/FluidFramework/pull/17590/files was what I was trying to reference.

But this looks good now.